charts/fleet/Chart.yaml

```diff
@@ -4,11 +4,11 @@ name: fleet
 keywords:
   - fleet
   - osquery
-version: v6.3.4
+version: v6.3.7
```

Comment:
@lukeheath and @rfairburn thoughts on this. latest release is https://github.com/fleetdm/fleet/blob/fleet-v4.64.0/charts/fleet/Chart.yaml#L7C7-L7C16 with 6.3.6 but this would be a downgrade in app version if they go from 6.3.6 -> this

Reply:
v6.4.0 is what is in main, and honestly I missed it but v4.64.0 should have published v6.4.1

https://github.com/fleetdm/fleet/blob/main/charts/fleet/Chart.yaml#L7

Reply:
we should make sure to re-publish v6.4.0 as 6.4.1 with the latest version of fleet to counteract that in the charts.

Reply:
We could also push this as v6.4.1 and make v4.64.0 as v6.4.2